apps/ledger-live-desktop/src/newArch/Collectibles/__integration__/nftsCollection.test.tsx

```diff
@@ -4,26 +4,36 @@
 import React from "react";
 import { render, screen } from "tests/testUtils";
 import { account, NftCollectionTest, NoNftCollectionTest } from "./shared";
+import mockedData from "./mockedData";
 
 jest.mock(
   "electron",
   () => ({ ipcRenderer: { on: jest.fn(), send: jest.fn(), invoke: jest.fn() } }),
   { virtual: true },
 );
 
+jest.mock("@ledgerhq/live-nft-react", () => {
+  return {
+    useNftMetadata: jest.fn(() => mockedData.mockReturnValue1),
+    isThresholdValid: jest.fn(() => true),
+    useNftGalleryFilter: jest.fn(() => mockedData.mockReturnValue2),
+    useNftCollectionMetadata: jest.fn(() => mockedData.mockReturnValue3),
+  };
+});
+
 describe("displayNftCollection", () => {
-  it("should display NFTs collection", async () => {
+  it("should display NFTs collections", async () => {
     render(<NftCollectionTest />, {
       initialState: {
         accounts: [account],
-        overriddenFeatureFlagsSelector: {
-          nftsFromSimplehash: true,
-        },
       },
       initialRoute: `/`,
     });
 
-    await expect(screen.getByText(/0x670fd103b1a08628e9557cd66b87ded841115190/i)).toBeVisible();
+    const mockedNftElements = screen.getAllByText(/mocked nft/i);
```

Comment:
maybe try with specific name ? to do screen.getByText ?

Reply:
it's caused by the mock since there is multiple nfts the mocked hook returns the same name for every nft